packages/core/src/errors.ts

```diff
@@ -0,0 +1,20 @@
+import { State, StateNode } from '.';
+
+export interface XStateErrorMeta {
+  /**
+   * The path the error occurred in
+   */
+  state?: State<any, any>;
```

Comment:
```suggestion
  state?: State<any, any, any, any>;
```

Probably need a few more `any`s to make TS happy unfortunately